src/main/scala/org/apache/spark/sql/delta/storage/BaseExternalLogStore.scala

```diff
@@ -208,9 +239,14 @@ abstract class BaseExternalLogStore(sparkConf: SparkConf, hadoopConf: Configurat
         throw e
```

Comment:
two lines above we catch Throwable, in other place we catch Exception, what is a correct way of catching all in scala/java?